monix-tail/shared/src/test/scala/monix/tail/IterantConcatSuite.scala

```diff
@@ -56,4 +56,26 @@ object IterantConcatSuite extends BaseTestSuite {
       (i :+ e) <-> (i ++ Iterant[Coeval].pure(e))
     }
   }
+
+  test("Iterant ++ Iterant rhs by name") { implicit s =>
```

Comment:
What do you think about renaming it to something that mentions that this is test for stack safety?

Reply:
Any propositions?

Reply:
"(...) is stack safe" :D

Reply:
Done.